samples/storage_csm_powerflex_v2110.yaml

```diff
@@ -183,10 +183,13 @@ spec:
     - name: authorization
       # enable: Enable/Disable csm-authorization
       enabled: false
+      # For PowerFlex Tech-Preview v2.0.0-alpha use v1.11.0 as configVersion DO NOT change the configVersion to v2.0.0-alpha
       configVersion: v1.11.0
       components:
         - name: karavi-authorization-proxy
+          # Use image: dellemc/csm-authorization-sidecar:v2.0.0-alpha for PowerFlex Tech-Preview v2.0.0-alpha
           image: dellemc/csm-authorization-sidecar:v1.11.0
+          # image: dellemc/csm-authorization-sidecar:v2.0.0-alpha
```

Comment:
Do we need two comments for the image?

Reply:
I wasn't sure so added the image line, may be will make easier for user or it is redundant